highway.tex

```diff
@@ -24,807 +26,1323 @@
+\begin{itemize}
+  \item {\bf unanimity.} $\vote(u) \geq B$ for all $u\in C_0$,
+  \item {\bf honesty.} $E(\sigma) \cap S(C_0) = \emptyset$,
+  \item {\bf convexity.} $u_0, u_2 \in C_i$ implies $u_1 \in C_i$, for all $u_0  \leq u_1 \leq u_2$ s.t. $S(u_1) = S(u_2) = S(u_3)$,
+  \item {\bf density.} $\big|S\big(\bar{D}(u) \cap C'_i)\big)\big| \geq q$ for all $u \in C_{i + 1}$.
+\end{itemize}
+Where $C'_i = \big\{u\in C_i \mid \text{ there exists } u'\in C_{i+1} \text{ s.t. } S(u) = S(u') \big\}$.
+\end{definition}
 
-Thus, once a validator has such a summit in their local protocol state $\sigma$, they know that eventually every honest validator will also see all of $\sigma$, and will only produce $p'$-messages with property $p$ from then on, unless a weight of at least $2t (1 - 2^{-k})$ validators equivocate. They can therefore consider $p$ to be \emph{finalized relative to $p'$}, with fault-tolerance threshold $2t (1 - 2^{-k})$.
+Intuitively, summits represent uninterrupted streaks of units produced by big subset of validators (i.e., a quorum) that vote for the same block $B$ --- a structure in DAG after which it is very unlikely that the votes will change, hence summits will be used to finalize blocks.
+In particular, as we will prove in the series of lemmas, checking the following condition will suffice to convince the validator that the given block $B$ will not be retracted, unless more than $t$ validators equivocates:
 
-Note that if we have at least a $2\ww(\mathcal{V})/3$-weight of honest validators, we could reasonably expect all of them to eventually contribute to a summit. If they persisted for enough rounds to achieve a high $k$, this would allow them to achieve a fault tolerance threshold of nearly $\ww(\mathcal{V})/3$. Thus, ignoring liveness considerations for the moment, we could hope that our system is secure against a third of the validators being faulty. In fact, in practice we might expect to be able to obtain summits with weight close to $\ww(\mathcal{V})$ if either there were few faulty validators at the time, or the faulty validators did not behave unusually. In this case, we might hope to obtain a fault-tolerance threshold of nearly everything.
+$$
+   \final(B, \sigma, t) \defeq \begin{cases}
+    1 &\mbox{if } \text{there exists $(q,k)$-summit for $B$ in $\sigma$ s.t. }(2q-n)(1-2^{-k})>t\\
+    0 &\mbox{otherwise}.
+    \end{cases}
+$$
 
+Now, based on the above definition we will refer to validator $V$ as having block $B$ \emph{finalized for confidence threshold} $t$ if $t$ is an integer value s.t. $\final(B,\sigma,t) = 1$, where $\sigma$ is $V$'s protocol state.
+Next we prove the key technical lemma, showing that after a summit of certain height and quorum size occurs, it is very unlikely for new units to vote against the vote of the summit. 
```

Comment:
I think it's safe to use a somewhat stronger formulation here, since the exact meaning gets formalized in this section anyway:
```suggestion
Next we prove the key technical lemma, showing that after a summit of certain height and quorum size occurs, new units will not vote against the vote of the summit.
```